gcc/rust/ast/rust-stmt.h

```diff
@@ -184,6 +184,8 @@ class ExprStmt : public Stmt
   Location locus;
 
 public:
+  virtual Location get_locus_slow () const override { return get_locus (); }
```

Comment:
Pretty pedantic, but this may optimise better with a `const final override` qualifier, since `get_locus_slow()` won't be overrided again at any later point. Also, since it isn't a new virtual method (it overrides the `Stmt` one), it doesn't need the `virtual` qualifier at the start.

Reply:
good point i will fix it before merge